overlays/default.nix

```diff
@@ -34,17 +70,12 @@ let
   rpi-kernels = builtins.foldl' (b: a: b // rpi-kernel a) { };
 in
 {
-  # disable firmware compression so that brcm firmware can be found at
-  # the path expected by raspberry pi firmware/device tree
-  compressFirmwareXz = x: x;
-  compressFirmwareZstd = x: x;
-
```

Comment:
Dropping these overrides would cause network issues, so dropping these from the overlay is probably the source of the network issues @SebTM mentions above.

Reply:
ooooo. Okay.